build-scripts/graal-env.js

```diff
@@ -17,7 +17,7 @@
 
 const GRAAL_OS = process.platform === 'win32' ? 'windows' : process.platform;
 const GRAAL_VERSION = process.env.GRAAL_VERSION || '20.1.0';
-const GRAAL_FOLDER = `graalvm-ce-java8-${GRAAL_OS}-amd64-${GRAAL_VERSION}`;
```

Comment:
Is making this change dependent on also switching from JDK8 to JDK11? Could you make the JDK switch in a separate PR?

Reply:
Yes of course - part of this change was proving to myself that I could upgrade this - but it's trivial to just use Java 8 for now.

Reply:
Actually no I can't. Building the windows native image on Java 8 can only be done with a very very old windows SDK (windows 7). Appveyor is the only build system I could find to support that - and it's extremely difficult to maintain.

GitHub actions allowed updating to java 11 for the main build, but java 8 for GWT. This massively simplified the build.